vpr/src/place/net_cost_handler.cpp

```diff
@@ -448,7 +453,88 @@ static double wirelength_crossing_count(size_t fanout);
 static void set_bb_delta_cost(const int num_affected_nets, double& bb_delta_c);
 
 /******************************* End of Function definitions ************************************/
+// Initialize the ts vectors
+void BB2D3DControlFlow::init(size_t num_nets, bool cube_bb_in) {
+    const int num_layers = g_vpr_ctx.device().grid.get_num_layers();
+
+    cube_bb = cube_bb_in;
```

Comment:
Could comment: we'll use either the 3D BB or per layer BB data structures, but not both.